hdfs-commons/src/main/java/org/apache/mesos/hdfs/config/HdfsFrameworkConfig.java

```diff
@@ -285,7 +286,11 @@ public int getConfigServerPort() {
   }
 
   public int getReconciliationTimeout() {
-    return getConf().getInt("mesos.reconciliation.timeout.seconds", DEFAULT_RECONCILIATION_TIMEOUT);
+    return getConf().getInt("mesos.reconciliation.timeout.seconds", DEFAULT_RECONCILIATION_TIMEOUT_SEC);
```

Comment:
@elingg: Consistent with what?  There is one line in this file which uses the timeout.sec convention 221.  Previous to this change there was also one instance of using timeout.seconds.  So that's a tie.  Please note that this change doesn't actually change what was already checked in.